Sources/_OpenAPIGeneratorCore/Translator/Responses/translateResponseHeader.swift

```diff
@@ -22,10 +22,18 @@ extension TypesFileTranslator {
     ///   - header: A response parameter.
     /// - Returns: A property blueprint.
     func parseResponseHeaderAsProperty(
+        responseKind: String = "",
```

Comment:
```suggestion
        responseKind: ResponseKind,
```

If the response kind is needed, pass it in as the typesafe type `ResponseKind` and don't provide a default value, we want to make sure all callers are updated.